### PR TITLE
docs: additional productionization notes

### PR DESCRIPTION
**Pull request by rspier**

### CODE REVIEW

File: doc/install-prod.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Additional Configuration
+
+Default fail2ban bantime is very short. Make it longer:
+
+```
+fail2ban-client start sshd
+fail2ban-client set sshd bantime 86400
+```
+
+Automatic security upgrades are a good idea, and probably outweight
```
Comment (wolfsage) [inline doc/install-prod.md:104]: nit: outweigh

File: doc/install-prod.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+the risks.
+
+```
+dpkg-reconfigure -plow unattended-upgrades
```
Comment (wolfsage) [inline doc/install-prod.md:108]: :sponge:

**Review by wolfsage — APPROVED** [review]
Seems good to me, thanks

**Comment by rjbs** [comment]
Thanks, applied!

**Pull request merged**